spring-cloud-kubernetes-fabric8-loadbalancer/src/main/java/org/springframework/cloud/kubernetes/fabric8/loadbalancer/Fabric8ServiceInstanceMapper.java

```diff
@@ -49,7 +50,7 @@ public class Fabric8ServiceInstanceMapper implements KubernetesServiceInstanceMa
 	}
 
 	@Override
-	public KubernetesServiceInstance map(Service service) {
+	public KubernetesAwareServiceInstance map(Service service) {
```

Comment:
Couldn't this be the interface?

Reply:
done, this is a weird refactor that intellij has done for me, I did not really notice it. good eye!